app/models/alchemy/essence_picture.rb

```diff
@@ -105,7 +105,7 @@ def thumbnail_url_options
       size = render_size || content.settings[:size]
 
       {
-        size: thumbnail_size(size, crop),
+        size: content.settings[:size] || thumbnail_size(size, crop),
```

Comment:
Will this not lead to ask the thumbnails being huge images?

Reply:
Not in my manual tests, but I will double check

Reply:
Yep, it does and removing it works as expected! 👍🏻